workitem/workitem_repository.go

```diff
@@ -588,6 +594,9 @@ func (r *GormWorkItemRepository) Create(ctx context.Context, spaceID uuid.UUID,
 		if err != nil {
 			return nil, errors.NewBadParameterError(fieldName, fieldValue)
 		}
+		if (fieldName == SystemAssignees || fieldName == SystemLabels) && fieldValue == nil {
```

Comment:
following-up with the comment above, I guess you could merge this case here with the one above, couldn't you ?

Reply:
also, could you add some tests at the repository level  to verify that the `assignees` and `labels` fields are removed when empty and not removed when not empty ?

Reply:
I will add tests in repo level. The above code is part of `Save` and here it is `Create` and the underlying data structure is different.

Reply:
Fixed dfb5f849d2b6b07cc94dc1ce1dd3a706075e44ab

Reply:
ok